src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { Comment } from '../../types/comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li>
+        <CommentInfo comment={comment} />
```

Comment:
i guess you should to add key={comment.id}

Reply:
👍